packages/redux/README.md

```diff
@@ -0,0 +1,11 @@
+# redux
+
+This library was generated with [Nx](https://nx.dev).
+
+## Building
+
+Run `nx build redux` to build the library.
+
+## Running unit tests
+
+Run `nx test redux` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
Should it be vitest? :thinking:

Reply:
This is a default readme, generated by Nx 🤷 

The project itself uses vitest indeed